lido_sdk/lido.py

```diff
@@ -88,7 +91,7 @@ def get_operators_keys(
                 "`get_operators_data` should be called first or provide `operators` param"
             )
 
-        self.keys = get_operators_keys(self._w3, operators)
+        self.keys = get_operators_keys(self._w3, operators, unused_keys_only)
```

Comment:
In this case we do not have all the keys to look for duplicates. It looks like we need to be able to merge the keys with the existing